### PR TITLE
Move nested urls to flat structure

### PR DESCRIPTION
**Pull request by asmacdo**
closes https://pulp.plan.io/issues/3160

### CODE REVIEW

**Comment by asmacdo** [comment]
@bmbouter, I'm not sure, but I am testing my guess by adding `six` to the pulpcore requirements in setup.py.

**Comment by asmacdo** [comment]
@bmbouter btw, that commit is just to test. IMO, we should get those string types some other way. We probably shouldn't keep six as a dep since we only support Python 2.

**Comment by asmacdo** [comment]
ok test

**Comment by werwty** [comment]
ok test

**Comment by werwty** [comment]
ok test

File: pulpcore/pulpcore/app/settings.py [hunk 1]
```diff
@@ -114,7 +114,7 @@
     'DEFAULT_FILTER_BACKENDS': ('django_filters.rest_framework.DjangoFilterBackend',),
     'DEFAULT_PAGINATION_CLASS': 'pulpcore.app.pagination.UUIDPagination',
     'PAGE_SIZE': 100,
-    'DEFAULT_PERMISSION_CLASSES': ('rest_framework.permissions.IsAuthenticated',),
```
Comment (werwty) [inline pulpcore/pulpcore/app/settings.py]: Don't forget to uncomment :)

**Comment by jortel** [comment]
Nice work, @asmacdo!  LGTM.

**Pull request merged**